packages/@jsii/integ-test/test/build-cdk.test.ts

```diff
@@ -41,16 +38,11 @@ describe('Build CDK', () => {
     const fileName = 'cdk.tar.gz';
     const tarFile = path.join(buildDir, fileName);
     const code = await downloadReleaseAsset(`https://api.github.com/repos/aws/aws-cdk/tarball/${release.data.tag_name}`);
-    const codeStream = fs.createWriteStream(tarFile);
 
-    // save to file and wait to finish
-    code.pipe(codeStream);
-    await new Promise(resolve => codeStream.on('close', () => {
-      resolve();
-    }));
+    await writeFileStream(code, tarFile);
 
     // unzip tar archive
-    await processes.spawn('tar', ['-xzvf', fileName], {
+    await processes.spawn('tar', ['-xzf', fileName], {
```

Comment:
I think there's a `tar` npm module of some sort we're using somewhere else. We should probably revisit later to stop shelling out here. But I don't reckon it's a blocker.

Reply:
yeah I noticed that earlier today, its an easy switch so gonna make the change.